.devcontainer/on-create.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+
+# Exit on any error
+set -e
+
+echo "🛠️  Running container setup (onCreate)..."
+
+# Enable corepack for package manager management
+echo "📦 Enabling corepack for package management..."
+if sudo corepack enable; then
+  echo "✅ corepack enabled successfully"
+else
+  echo "❌ Failed to enable corepack"
+  exit 1
+fi
+
+# Install latest pnpm using corepack
+echo "📦 Installing latest pnpm using corepack..."
+if sudo corepack prepare pnpm@latest --activate; then
```

Comment:
Rather than installing the latest, `packageManager` field in the root `package.json` should already be respected by corepack. Could be useful to somehow eagerly install it here though?